lib/class/xml_data.class.php

```diff
@@ -221,18 +221,19 @@ private static function playlist_song_tracks_string($song, $playlist_data)
     public static function keyed_array($array,$callback='')
     {
         $string = '';
-
+        return $array;
```

Comment:
I didn't notice that, but yes... we shouldn't merge my xml_data changes if I made any.